apps/admin-page/src/app/components/layout/layout.component.css

```diff
@@ -0,0 +1,69 @@
+.navbar {
+  height: var(--navbar-height);
+  padding-right: 12.8px;
+  padding-left: 12.8px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--main-color);
+  color: var(--font-color);
+}
+
+.navbar .navbar-btn {
```

Comment:
The navbar buttons should have a `hover` effect to increase the visual feedback when the user is interacting with the buttons.
The hover effect should be a change of colour to lighter blue with a slight transition effect